notebook2/notebook/static/styles/buttons.scss

```diff
@@ -0,0 +1,48 @@
+@import './variables/colors.scss';
+#theme-site {
+  a {
+    color: $black;
+    text-decoration: none;
+    &.active,
+    &:hover {
+      color: $devil-gray;
+    }
+  }
+  button {
+    outline: none;
+    /* border: none; */
+    /* display: inline-block; */
+    /* width: 20px; */
+    border: 1px solid $black;
+    background-color: $white;
+    padding: 10px;
+    border-radius: 3px;
+    &:focus,
+    &:hover {
+      border-color: $devil-gray;
+    }
+  }
+  .link-button,
+  button {
+    cursor: pointer;
```

Comment:
isn't this the default for anchors and buttons?

Reply:
.link-button can apply to elements other than button.

Reply:
@danking, actually, double checked, button does not default to pointer either, at least not in chrome or safari.